src/plugins/undoRedo/undoRedo.js

```diff
@@ -44,6 +44,10 @@ function UndoRedo(instance) {
       return;
     }
 
+    arrayEach(changes, (change) => {
+      change[1] = instance.propToCol(change[1]);
```

Comment:
Changing `prop`=>`col` by reference? 🤔
I think you can prepare map `prop`=>`col` in L51.